ivadomed/config_manager.py

```diff
@@ -65,7 +66,7 @@ def load_json(config_path):
 
 
 # To ensure retrocompatibility for parameter changes in configuration file
-KEY_CHANGE_DICT = {'UNet3D': 'Modified3DUNet'}
+KEY_CHANGE_DICT = {'UNet3D': 'Modified3DUNet', 'bids_path': 'path_data', 'log_directory': 'path_output'}
```

Comment:
Cool I didn't know we had this retrocompatibility feature!

Reply:
Yes you're right! We should definitely keep this in mind!

Reply:
https://github.com/ivadomed/ivadomed/issues/670